doc/source/drivers/raster/s102.rst

```diff
@@ -0,0 +1,65 @@
+.. _raster.s102:
+
+================================================================================
+S102 -- S-102 Bathymetric Surface Product
+================================================================================
+
+.. shortname:: S102
+
+.. build_dependencies:: libhdf5
+
+.. versionadded:: 3.8
+
+This driver provides read-only support for bathymetry data in the S-102 format,
+which is a specific product profile in an HDF5 file
+
+S-102 files have two image bands representing depth (band 1),
+uncertainty (band 2) values for each cell in a raster grid area.
+
+Note that positive values of depth mean values *below* the reference surface
+of the vertical datum. The :oo:`DEPTH_OR_ELEVATION` open option can be set
+to ``YES`` to expose depth values as elevation values, by negating their sign
+(i.e. positive values of elevation mean values above the reference surface)
```

Comment:
DEPTH_OR_ELEVATION=YES??? Did you mean DEPTH_AS_ELEVATION?

Reply:
ah, I meant DEPTH_OR_ELEVATION=ELEVATION . I'm not super happy with the naming of that option, but basically it is a way for the user to express if it wants depth or elevation, as the name implies :-)